src/views/v2/commerce/checkout/cart.php

```diff
@@ -13,20 +13,21 @@
  *
  * @version 5.1.9
  *
- * @var \Tribe__Template $this                  [Global] Template object.
- * @var Module           $provider              [Global] The tickets provider instance.
- * @var string           $provider_id           [Global] The tickets provider class name.
- * @var array[]          $items                 [Global] List of Items on the cart to be checked out.
- * @var bool             $must_login            [Global] Whether login is required to buy tickets or not.
- * @var string           $login_url             [Global] The site's login URL.
- * @var string           $registration_url      [Global] The site's registration URL.
- * @var bool             $is_tec_active         [Global] Whether `The Events Calendar` is active or not.
- * @var int              $section               Which Section that we are going to render for this table.
- * @var array[]          $gateways              [Global] An array with the gateways.
- * @var int              $gateways_active       [Global] The number of active gateways.
+ * @var \Tribe__Template $this             [Global] Template object.
+ * @var Module           $provider         [Global] The tickets provider instance.
+ * @var string           $provider_id      [Global] The tickets provider class name.
+ * @var array[]          $items            [Global] List of Items on the cart to be checked out.
+ * @var bool             $must_login       [Global] Whether login is required to buy tickets or not.
+ * @var string           $login_url        [Global] The site's login URL.
+ * @var string           $registration_url [Global] The site's registration URL.
+ * @var bool             $is_tec_active    [Global] Whether `The Events Calendar` is active or not.
+ * @var int              $section          Which Section that we are going to render for this table.
+ * @var array[]          $gateways         [Global] An array with the gateways.
+ * @var int              $gateways_active  [Global] The number of active gateways.
  */
 
 $post = get_post( $section );
+$user = wp_get_current_user();
```

Comment:
Let's move the `$user` to billing details, as we don't use it here. We should get that only if we're running the template.

Reply:
Hmm. This must have been something that was done in one of Ricardo's tickets. I'll move it when I get to the checkout tickets. I'll add a `@todo`